test/common/statHelpers.test.js

```diff
@@ -0,0 +1,58 @@
+var sinon = require('sinon');
+var chai = require("chai")
+chai.use(require("sinon-chai"))
+var expect = chai.expect
+
+var shared = require('../../common/script/index.js');
+
+describe('helper functions used in stat calculations', function() {
+
+  var HEALTH_CAP = 50;
+  var LEVEL_CAP = 100;
+  var LEVEL = 57;
+  var BONUS = 600;
+  var MAXIMUM = 200;
+  var HALFWAY = 75;
+
+  it('provides a maximum Health value', function() {
+    expect(shared.maxHealth).to.eql(HEALTH_CAP);
+  });
+
+  describe('maximum level cap', function() {
+    it('returns a maximum level for attribute gain', function() {
+      expect(shared.maxLevel).to.eql(LEVEL_CAP);
+    });
+
+    it('returns level given if below cap', function() {
+      expect(shared.capByLevel(LEVEL)).to.eql(LEVEL);
+    });
+
+    it('returns level given if equal to cap', function() {
+      expect(shared.capByLevel(LEVEL_CAP)).to.eql(LEVEL_CAP);
+    });
+
+    it('returns level cap if above cap', function() {
+      expect(shared.capByLevel(LEVEL_CAP + LEVEL)).to.eql(LEVEL_CAP);
+    });
+  });
+
+  describe('Experience to next level', function() {
+    it('increases Experience target from one level to the next', function() {
+      expect(shared.tnl(LEVEL + 1)).to.be.greaterThan(shared.tnl(LEVEL));
+    });
+  });
+
+  describe('diminishing returns', function() {
```

Comment:
Let's use the arrow function syntax: `() =>` instead of `function()`.

anonymous `function ()` should only be used if we need to reference `this` in the function.